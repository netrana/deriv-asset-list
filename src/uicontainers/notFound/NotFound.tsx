import React from 'react';

import { Result } from 'components/helpers';

const NotFound = () => {
  return (
    <Result
      status="404"
      title="404"
      subTitle="Sorry, the page you visited does not exist."
    />
  );
};

export default NotFound;
