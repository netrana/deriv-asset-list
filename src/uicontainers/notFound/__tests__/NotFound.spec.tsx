import React from 'react';
import render from 'react-test-renderer';

import NotFound from '../NotFound';

jest.mock('components/helpers', () => {
  const actual = jest.requireActual('components/helpers');
  return {
    ...actual,
    Result: 'Result',
  };
});

describe('NotFound', () => {
  describe('Snapshots', () => {
    it('renders correctly', () => {
      const component = render.create(<NotFound />);
      expect(component).toMatchSnapshot();
    });
  });
});
