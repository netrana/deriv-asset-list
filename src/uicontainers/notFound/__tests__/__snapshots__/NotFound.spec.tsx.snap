// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Snapshots renders correctly 1`] = `
<Result
  status="404"
  subTitle="Sorry, the page you visited does not exist."
  title="404"
/>
`;
